### DISCUSSION TITLE
Integrate ExCheck

### DISCUSSION THREAD

**Discussion by byhbt** [nimblehq/elixir-templates#304] [RFCs]
## Why

`ex_check` (https://hexdocs.pm/ex_check) is an all-in-one command, which helps developers run the common checks for Elixir Application within 1 command. Supporting checks are listed in this document https://hexdocs.pm/ex_check/Mix.Tasks.Check.html#module-tools.

It's quite handy when the developers only need to remember one check command to execute: `mix check`, instead of remembering multiple separate commands and execute it manually.

## Who Benefits?

Elixir Developers

**Comment by andyduong1920** [comment]
I voted `No` as

1. We had the `mix codebase` and `mix codebase.fix` which contained all checks for our code
2. We don't need to add 1 more 3rd library into the app

**Comment by andyduong1920** [comment]
@byhbt any update on this RFC
